src/js/fc.js

```diff
@@ -12,6 +12,7 @@ var LED_COLORS;
 var LED_MODE_COLORS;
 var PID;
 var PID_names;
+var MSP_PIDs;
```

Comment:
Can we either use CamelCase or SNAKE_CASE, but not a mix of the two please?

Reply:
I'm happy to change this if you want, but this casing was used to match the pre-existing `PIDs` variable which it shadows.

Reply:
Using clean patterns is preferable over being consistent with the use of broken patterns.

Reply:
Fair point, I've change the casing for this variable.